#!/usr/bin/env python
# encoding: utf-8

# host = '10.20.16.207'
# port = 8523
# thrift_file = 'idl/ad/arbiter.thrift'
# module_name = 'hi_thrift'
# service_name = 'AdArbiterService'
# api_name = 'GenFilter'
# req_class_name = 'GenFiltersRequest'
# req_data_file = 'req.json'
# resp_class_name = 'GenFiltersResponse'

include_dirs = ['.']
include_dir = 'idl'
#
host = '127.0.0.1'
port = 1111
thrift_file = 'hi.thrift'
module_name = 'hi_thrift'
service_name = 'HiService'
api_name = 'Hi'
req_class_name = 'HiRequest'
req_data_file = 'hi.json'
resp_class_name = 'HiResponse'

